gradle/render-javadoc.gradle

```diff
@@ -17,11 +17,15 @@
 
 // generate javadocs by using Ant javadoc task
 
+// utility function to convert project path to document output dir
+// e.g.: ':lucene:analysis:common' => 'analysis/common'
+def pathToDocdir = { path -> path.split(':').drop(2).join('/') }
+
 allprojects {
   plugins.withType(JavaPlugin) {
     ext {
       javadocRoot = project.path.startsWith(':lucene') ? project(':lucene').file("build/docs") : project(':solr').file("build/docs")
-      javadocDestDir = "${javadocRoot}/${project.name}"
+      javadocDestDir = "${javadocRoot}/" + pathToDocdir(project.path)
```

Comment:
You can call the function inside the evaluated string too; "${javadocRoot}/${pathToDocDir(project.path)}"

Reply:
Fixed, thanks.